src/wasm-interpreter.h

```diff
@@ -1268,12 +1268,11 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
       return flow;
     }
     NOTE_EVAL1(curr->event);
-    auto exn = std::make_unique<ExceptionPackage>();
-    exn->event = curr->event;
+    ExceptionPackage exn(curr->event);
     for (auto item : arguments) {
-      exn->values.push_back(item);
+      exn.values.push_back(item);
     }
-    throwException(Literal(std::move(exn)));
+    throwException(Literal::makeExn(exn));
```

Comment:
Another refactor on my hunt for memory corruption, trying to hide the pointer stuff from calling code and let literals manage these. Seemed like a good thing to do semantically so I kept it, but apart from that it involves at least one more copy than needed I guess.